test/Combinatorics/Matroids/Matroids.jl

```diff
@@ -338,5 +338,10 @@
         @test is_minor(N,NN)
         @test !is_minor(NN,N)
         @test !is_minor(fano_matroid(), uniform_matroid(2,4))
-   end
+    end
+
+    @testset "matroid automorphism" begin
+        M = uniform_matroid(3, 5)
+        @test automorphism_group(M) == 
```

Comment:
Something is missing here.

Reply:
The implementation is not entirely correct (wrong action), and it does not reuse enough code that already exists.  @lkastner is on it.